lib/AppConsole.h

```diff
@@ -42,7 +42,11 @@
 #define APPCONSOLE_H
 
 #define CON_MAX_STRINGLEN 768
+
+bool con_Create(int flags);
```

Comment:
are these new declarations also implemented in the Windows build anywhere? If they're not common to both builds, they should probably not go here in a common header

Reply:
Yes, these supposed to go to lnxapp.h... Still, in #410 they will be deleted anyway.

Reply:
On second thought I decided don't move these declarations at all.